docs/guides/guide-vmanomaly-vmalert.md

```diff
@@ -0,0 +1,403 @@
+# VmAnomaly Quickstart
+
+### Prerequisites
+- In this tutorial it is assumed that you have VictoriaMetrics already, and you are familiar with VictoriaMetrics ecosystem:
+  -  [victoriametrics](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html)
+  -  [vmalert](https://docs.victoriametrics.com/vmalert.html)
+  -  [vmagent](https://docs.victoriametrics.com/vmagent.html)
+  
+- It is assumed that you are familiar with Grafana and Docker
+## What is VmAnomaly?
+*VictoriaMetrics Anomaly Detection* is a service that continuously scans VictoriaMetrics time series and detects unexpected changes within data patterns in real-time. It does so by utilizing user-configurable machine learning models.
+
+All the service parameters (model, schedule, input-output) are defined in a config file.
```

Comment:
By this moment, the reader is unaware of the `schedule` or `input-output` concepts. So this sentence doesn't bring any additional knowledge to him/her

Reply:
deleted from here